DOC-1256 clarify status of zoning and offloading (#1285)
spatialos/UnrealGDK: SpatialGDK/Documentation/content/technical-overview/gdk-concepts.md

| RPC | SpatialOS component’s command or event | A type of data stored in a SpatialOS component. Note that a SpatialOS component’s command is **not** the same as an Unreal command. |
| Owning connection | EntityACL component | |
| Conditional property replication | Dynamic `component_delivery` filter | |
| Server | Server-worker instance | You can have one or more server-worker instances running the cloud element of your game. |

You can find out more about [entities]({{urlRoot}}/content/glossary#entity), SpatialOS [components]({{urlRoot}}/content/glossary#spatialos-component) and their properties, commands, and events, as well as [server-workers]({{urlRoot}}/content/glossary#worker), in the [glossary]({{urlRoot}}/content/glossary).

## GDK for Unreal concepts
We’ve introduced some new concepts to facilitate the fact that SpatialOS enables you to spread computation between multiple servers - known as “server-worker instances” in SpatialOS.

### Offloading

> Offloading is one of the multiserver options for working with SpatialOS. It involves allocating the authority over specific groups of Actors to a server-worker instance other than your main Unreal server-worker instance. 

Reviewer: I think this doesn't really explain what offloading is.
Have made a suggestion elsewhere on how to explain what it is.

For example, you could offload computationally expensive but latency-tolerant systems, such as advanced AI or large-scale background physics simulation, so that your main server-worker instance can run other systems at a larger scale.

> <%(Lightbox image="{{assetRoot}}assets/offloading-diagram.png")%>

Reviewer: This diagram makes me sob. It needs a jira ticket, I think?

Author: Hmm, yeah maybe worth making a separate ticket (from the offloading doc review one). I'll do that now.

_Offloading: The offloaded server-worker instance has authority only over Red Actors, and the main Unreal server-worker instance that runs major game systems has authority over all other Actors._

For more information, see [Offloading overview]({{urlRoot}}/content/workers/offloading-concept).

### Actor groups
To facilitate offloading, we've created the concept of Actor groups to help you configure the Actor types that a given type of server-worker instance has authority over. In the Unreal Editor, you can create Actor groups, assign Actor classes to a group, and then assign each group to a server-worker type.

> Before you start to use offloading in your game, make sure that you’re familiar with the [best practices]({{urlRoot}}/content/workers/offloading-concept#best-practices) and with how to [set up your game feature for offloading]({{urlRoot}}/content/workers/set-up-offloading).

Reviewer: This is a bit bonkers, as no-one's going to try offloading without actually finding out about it first, or are they?
We usually say "for guidance" or "for more information".
```suggestion
For guidance on offloading, see the documentation on offloading [best practices]({{urlRoot}}/content/workers/offloading-concept#best-practices) and [how to set up your game for offloading]({{urlRoot}}/content/workers/set-up-offloading).
```
Totally gonna rename that title so renaming it here.

Reviewer: >This is a bit bonkers, as no-one's going to try offloading without actually finding out about it first, or are they?
See how I thought, Naaah, no-one would and then thought - maybe they are!

### Zoning

> Like [offloading](#offloading), zoning is a multiserver option for working with SpatialOS. It involves splitting up the world into zones, known as “areas of authority”, with a different server-worker instance responsible for each. A server-worker instance can make updates only to Actors that are in its area of authority.

Reviewer: This could suggest that offloading and zoning are similar.
Also it's probs worth saying it's about sharing the server computation load ?
Also, we should probs never say "zones" and only ever "areas" because that's a synonym happening.

```suggestion
Zoning is another of the multiserver options when working with SpatialOS, it works differently to [offloading](#offloading). 
Zoning involves sharing the server computation load by splitting up the world into areas of authority, with a different server-worker instance responsible for the computation of each area. A server-worker instance can make updates only to Actors that are in its area of authority.
```

For more information, see the documentation on [Singleton Actors]({{urlRoot}}/content/singleton-actors).

### Actor throttling

> We have introduced a concept of 'Actor throttling' in order to optimize server performance when using the Unreal GDK, it can be accessed via the [Runtime Settings]({{urlRoot}}/content/unreal-editor-interface/runtime-settings). With the Unreal GDK it is possible to have multiple servers computing on the same game world, this means that no one server could have a full view of the entire world at once. Additionally, servers in the Unreal GDK replicate actors only to a single connection to SpatialOS rather than to each individual client as in native Unreal. To accommodate this, in order to keep the state of the SpatialOS simulation accurate, all actors which are in a particular server's view are marked as relevant for replication on each frame. This can however cause performance issues if there are say, 3000 actors on a single server. 

Reviewer: Same comment as before about chatty blogpost style text.
Shall we make a ticket for this in the fallout tasks because this is signficant?

Author: As above, non-tech-writered (and I've already created a ticket :) https://improbableio.atlassian.net/browse/DOC-1265

The concept of Actor Throttling allows you to set a simple limit on the number of actors which can be replicated to SpatialOS in each frame and can help boost server performance. The number set in the throttle will dictate how many actors can be replicated, with the highest priority actors (using Unreal's actor priority system) being replicated first.

We recommend using Actor Throttling with a combination of Unreal's Net Update Frequency, as this will limit the number of actors which are added to the 'ConsiderList' for replication in order to optimize your servers performance. Experiment with the limit by setting it high and then lowering over time until you see client latency issues in a realistic game scenario. 

## Non-Unreal computation

> By default, the GDK uses a single Unreal server-worker type (which is like a mold for a server-worker instance) to handle all server-side computation. However, you can set up additional server-worker types that do not use Unreal or the GDK. 

Author: Thanks! I think it should be clear what "which" is referring to, but happy to take your updates, especially as this is an early page.

> By default, the GDK uses a single Unreal server-worker type (which is like a mold for a server-worker instance) to handle all server-side computation. However, you can set up additional server-worker types that do not use Unreal or the GDK. 

Reviewer: It reads like "a single Unreal server-worker type" is like a mold. Rather than "type" is a mold.
